tests/test_include_verilog.py

```diff
@@ -0,0 +1,25 @@
+import fault
+import magma as m
+
+
+def test_include_verilog():
+    with open("tests/cells_sim.v") as f:
```

Comment:
probably should test this feature with a different/simple file, to avoid including a big verilog file.